GliaWidgets/Sources/Theme/Theme.Text.Accessibility.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+extension Theme.Text {
+    /// Accessibility properties for Text.
+    public struct Accessibility: Equatable {
+        /// Flag that provides font dynamic type by setting `adjustsFontForContentSizeCategory` 
+        /// for component that supports it.
+        public var isFontScalingEnabled: Bool
+
+        /// - Parameters:
+        ///   - isFontScalingEnabled: Flag that provides font dynamic type by setting
+        ///    `adjustsFontForContentSizeCategory` for component that supports it.
```

Comment:
do we need to have additional `///` here?

Reply:
fixed [14839cf](https://github.com/salemove/ios-sdk-widgets/pull/887/commits/14839cfb8b707e3a12197ff4bd96b79b48073da5)